Onepunch/changelog.md

```diff
@@ -0,0 +1,28 @@
+## Versions
```

Comment:
Not really important but I think it's visually more clear and pleasant to use checkbox with completed features and missing ones (respectively checked or not) instead of `code` blocks.

So maybe 
- [x] A few optimizations.
- [ ] New Day/Night selection after spicetify update.
- [ ] Rework the white theme.

Instead of

```
		$ A few optimizations.
		$ New Day/Night selection after spicetify update.
		$ Rework the white theme.
```